doc/_ext/nope/path/subpaths.py

```diff
@@ -0,0 +1,25 @@
+from pynopegl_utils.misc import SceneCfg, scene
+
+import pynopegl as ngl
+
+
+@scene()
+def subpaths(cfg: SceneCfg):
+    cfg.aspect_ratio = (1, 1)
+
+    keyframes = [
+        ngl.PathKeyMove(to=(0, 1, 0)),
+        ngl.PathKeyBezier3(to=(3, 0, 0), control1=(1, 3, 0), control2=(3, 2, 0)),
+        ngl.PathKeyBezier2(to=(0, -3, 0), control=(3, -2, 0)),
+        ngl.PathKeyBezier2(to=(-3, 0, 0), control=(-3, -2, 0)),
+        ngl.PathKeyBezier3(to=(0, 1, 0), control1=(-3, 2, 0), control2=(-1, 3, 0)),
+        ngl.PathKeyMove(to=(0, -1, 0)),
+        ngl.PathKeyLine(to=(-3, -3, 0)),
+        ngl.PathKeyMove(to=(-2.5, -2, 0)),
+        ngl.PathKeyLine(to=(-3, -3, 0)),
+        ngl.PathKeyLine(to=(-2, -3, 0)),
+        ngl.PathKeyClose(),  # close the sub-path (arrow tip triangle)
```

Comment:
The arrow fletching is missing !

Reply:
Changed that example to something different

Reply:
Thanks.